src/main/io/osd.c

```diff
@@ -1346,6 +1350,16 @@ static void osdUpdateStats(void)
         }
     }
 #endif
+#ifdef USE_ESC_SENSOR
```

Comment:
Should also check that the ESC_SENSOR feature is enabled. `featureIsEnabled(FEATURE_ESC_SENSOR)`.